src/sentry/tasks/groupowner.py

```diff
@@ -94,10 +103,10 @@ def process_suspect_commits(event, **kwargs):
                 except Commit.DoesNotExist:
                     logger.info(
                         "process_suspect_commits.skipped",
-                        extra={"event": event.event_id, "reason": "no_commit"},
+                        extra={"event": event_id, "reason": "no_commit"},
                     )
                 except Release.DoesNotExist:
                     logger.info(
                         "process_suspect_commits.skipped",
-                        extra={"event": event.event_id, "reason": "no_release"},
+                        extra={"event": event_id, "reason": "no_release"},
```

Comment:
Considering leaving out event_id from this log so we don't have to pass it.

Reply:
I think it's fine to pass, compared to the stack trace it's not much data.